x-pack/plugins/session_view/public/components/process_tree_node/styles.ts

```diff
@@ -50,7 +50,7 @@ export const useStyles = ({ depth, hasAlerts }: StylesDeps) => {
       let borderColor = 'transparent';
 
       // TODO: alerts highlight colors
```

Comment:
Are we sure we only want to do this for investigated alert? I think in the mock up it shows the red left border for all alerts, maybe just investigated has the background color set?

e.g 
![image](https://user-images.githubusercontent.com/16198204/157313010-62011a37-3164-48f3-888e-9bad03ee2be2.png)

Reply:
![image](https://user-images.githubusercontent.com/16198204/157313184-aee3da9e-5236-467e-8043-6b4772e2c84d.png)

Reply:
Yea, I changed it in this PR to just having the investigated to have the background color. I haven't done the red left border yet, will be working on that today.

Reply:
seems like the border was there but the calculation is outdated after some updates to the process tree indent, my fix will just be correcting the correct negative margin here for marginLeft